__docs__/wonder-blocks-icon-button/icon-button.stories.tsx

```diff
@@ -167,14 +168,57 @@ export const Variants: StoryComponentType = {
 };
 
 /**
- * An IconButton on a dark background.
- * Only the primary kind is allowed to have the `light` prop set to true.
+ * IconButton has a `color` that is either `default` (the default, as shown
+ * above) or `destructive` (as can seen below):
+ */
+export const WithColor: StoryComponentType = {
```

Comment:
**note:** I noticed that we don't have docs for the `color` prop, so I added a new story to cover that.